src/map/mob.cpp

```diff
@@ -3264,6 +3336,7 @@ int32 mob_dead(struct mob_data *md, struct block_list *src, int32 type)
 		add_timer(tick + (!battle_config.delay_battle_damage ? 500 : 0), mob_delay_item_drop, md->bl.id, 0);
 	}
 
+	// MVP Reward
 	if( mvp_sd && md->get_bosstype() == BOSSTYPE_MVP ){
```

Comment:
```suggestion
	if( mvp_sd != nullptr ){
```
Bosstype check is included in getmvp already.